### PR TITLE
Filter xcodebuild's output

### PR DESCRIPTION
**Pull request by chinhodado**
xcodebuild's output is incredibly verbose. This will grep out only lines with error/warning.

Before: [~15,000 lines](https://s3.amazonaws.com/archive.travis-ci.org/jobs/64956564/log.txt)
After:  [~500 lines](https://api.travis-ci.org/jobs/64957057/log.txt?deansi=true)

### CODE REVIEW

File: .travis-build.sh [hunk 1]
```diff
@@ -18,5 +18,5 @@ elif [ "$TRAVIS_OS_NAME" = "osx" ]; then
     export Qt5_DIR=$(brew --prefix)/opt/qt5
     mkdir build && cd build
     cmake .. -GXcode
-    xcodebuild -configuration Release
+    xcodebuild -configuration Release | egrep '^(/.+:[0-9+:[0-9]+:.(error|warning):|fatal|===)'
```
Comment (neobrain) [inline .travis-build.sh]: comment plz.

Comment (chinhodado) [inline .travis-build.sh]: You mean explaining the regex, what it matches etc.?

**Comment by archshift** [comment]
I'm not sure this is the best approach. How about one of the many tools to prettify xcodebuild's output?

**Comment by chinhodado** [comment]
I only know xcpretty, and it doesn't support displaying warnings.

**Comment by archshift** [comment]
What about using `xctool` instead of `xcodebuild`?

**Comment by archshift** [comment]
Also apparently xcpretty [supports warnings now](https://github.com/supermarin/xcpretty/issues/108#issuecomment-107734863).

**Comment by chinhodado** [comment]
Cool, I'll close this one then.

**Pull request closed without merging**